src/components/titlePanel/styles/titlePanel.css

```diff
@@ -0,0 +1,31 @@
+@import '../../../styles/variables.css';
+
+:root {
+	--title-panel-border-width: 1px;
```

Comment:
I think these are likely standard component variables / styling and thus should go to `variables.css` (with different names)